src/main/java/com/dnsimple/data/ZoneDistribution.java

```diff
@@ -0,0 +1,13 @@
+package com.dnsimple.data;
+
+import com.google.api.client.util.Key;
+
+public class ZoneDistribution {
+  @Key("distributed")
+  private boolean distributed;
+
+  public boolean getDistributed() {
```

Comment:
in Java-land you would likely call this method `isDistributed()`.

Reply:
Right. It's been a long time, I forgot about this. 😉